src/main/java/org/jenkins/plugins/lockableresources/queue/LockableResourcesStruct.java

```diff
@@ -134,5 +136,8 @@ public String toString() {
       + this.requiredNumber;
   }
 
-  private static final long serialVersionUID = 1L;
+  private Object readResolve() {
```

Comment:
for this I need some java programmer. I have no idea if I do it in correct way.

Reply:
@gounthar  has you any idea?

Reply:
What are you trying to achieve?
What do you think of a short meeting so that we can pair (not that I'm any good at Java anymore, but...)?

Reply:
> What are you trying to achieve? What do you think of a short meeting so that we can pair (not that I'm any good at Java anymore, but...)?

A call. LGTM. It will be nice to meet you ;-) But next week. This weekend will be more or less digital detox for me.